modules/lm-coursier/src/main/scala/lmcoursier/definitions/Authentication.scala

```diff
@@ -1,12 +1,14 @@
 package lmcoursier.definitions
 
-import dataclass.data
+import dataclass._
 
 @data class Authentication(
   user: String,
   password: String,
   optional: Boolean = false,
-  realmOpt: Option[String] = None
+  realmOpt: Option[String] = None,
+  @since
+  headersOpt: Option[Seq[(String,String)]] = None
 ) {
   override def toString(): String =
     withPassword("****")
```

Comment:
I think `toString` needs to be updated, to hide the header values (that can contain secrets).